doc/source/examples/similarity_explanations_mnist.ipynb

```diff
@@ -0,0 +1,583 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        preds_sim = model(X_sim).numpy().argmax(axis=1)
</pre><p>There is no point in calling the model on the entire dataset.</p><p>Please consider calling the model only on the top k predictions. This can easily be achieved by using</p><p><code>expls.data['ordered_indices'][j][:k]</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>This is needed to produce the plots showing the scores averaged over each predicted class </p>